src/fu-self-test.c

```diff
@@ -1767,8 +1781,13 @@ fu_common_firmware_builder_func (void)
 						    "startup.sh",
 						    "firmware.bin",
 						    &error);
-	g_assert_no_error (error);
-	g_assert (firmware_blob != NULL);
+	if (firmware_blob == NULL) {
+		if (errno == EPERM) {
```

Comment:
Can't we check the `@error` here rather than `EPERM`?

Reply:
Fixed